### ISSUE TITLE
Stop opening a code editor on error. 

### ISSUE THREAD

**Issue by JoueBien** [vercel/next.js#14488]
# Bug report

## Describe the bug
Every time I have an error I get a prompt to open the file where the error occurred. There isn't any way to turn this off. 

In my case it opens the UAC prompt because sometimes I have to run vs code in an elevated mode. It would be 1000 times more annoying to lose my current location in the files I have open in VS code if I wasn't running VS code in an elevated mode (which gives me an option to say do not open).

## To Reproduce

Create something that will cause an error during live reloading. The running process will attempt to open the file where the error is.

## Expected behavior

Don't open anything unless it's passed as a parameter via the command line. 

## Screenshots
What happens after click don't open in UAC.
![image](https://user-images.githubusercontent.com/7506155/85375484-b32d9280-b579-11ea-8584-1652dc42a500.png)

## System information

- OS: [Windows]
- Browser n/a
- Version of Next.js: [e.g. 9.3.0]
- Version of Node.js: [e.g. 12.14.0]

**Comment by timneutkens** [comment]
The editor only opens when you click the link to the error location 🤔 Can you provide a video or more details as this report is not super clear about what you're experiencing.

**Comment by JoueBien** [comment]
I'm experiencing next opening the code editor as soon as a JS error occurs - instead of it waiting until I click the link on the error location.

**Comment by timneutkens** [comment]
> I'm experiencing next opening the code editor as soon as a JS error occurs - instead of it waiting until I click the link on the error location.

Please provide a full reproduction as asked for

**Comment by Timer** [comment]
Closing as no reproduction provided.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
